### PR TITLE
feat: [CI-2533]: zero state ci project overview 

### PR DESCRIPTION
**Pull request by mtran7**
### Summary

New zero state screens and call to actions for CI:
#### Screenshots
BEFORE: 
image
AFTER:
image
image

No regression for CD:

image

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 17 Issues 
* Code issue 17 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10306)

**Review by udayvunnam — APPROVED** [review]

**Comment by udayvunnam** [comment]
perfect!

File: src/modules/75-ci/pages/dashboard/CIDashboardPage.tsx [hunk 1]
```diff
@@ -78,6 +128,22 @@ export const CIDashboardPage: React.FC = () => {
     }
   })
 
+  // this for detecting whether the project have any pipelines or if pipelines then any execution or not
```
Comment (d1wilko) [inline src/modules/75-ci/pages/dashboard/CIDashboardPage.tsx:131]: I find this sentence confusing

**Review by d1wilko — CHANGES_REQUESTED** [review]
Just 1 small issue - other than that it looks good to go :)

**Review by vardanbansal-harness — APPROVED** [review]

File: src/modules/75-ci/pages/dashboard/CIDashboardPage.tsx (hunk 1 above)
Comment (mtran7) [reply src/modules/75-ci/pages/dashboard/CIDashboardPage.tsx:131]: agreed - removed as it's not a necessary comment

**Comment by mtran7** [comment]
retrigger jest

**Review by udayvunnam — APPROVED** [review]

**Review by d1wilko — APPROVED** [review]
LGTM

**Pull request merged**